modules/ytp-assets-common/src/less/components/layout.less

```diff
@@ -36,6 +36,14 @@ body {
   margin: 0 auto;
   max-width: 1140px;
   padding: 0 15px;
+  border-radius: 0px;
+  border: 1px solid @color--extra-light-grey;
+  background: #ffffff;
+}
+
+.search-dataset-wrapper{
+    border: none!important;
+    background: #fafafa!important;
```

Comment:
`!important` always feels like something is wrong. Could the priority of the rule be increased by increasing specifity instead?

Reply:
Same in `sidebar.less`